Proofs/AES-GCM/AESGCMEncKernelSym.lean

```diff
@@ -0,0 +1,56 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Yan Peng
+-/
+import Proofs.«AES-GCM».AESGCMEncKernelPre
+import Tactics.Sym
+import Tactics.StepThms
+import Correctness.ArmSpec
+import Arm.Memory.Separate
```

Comment:
```suggestion
import Proofs.«AES-GCM».AESGCMEncKernelPre
import Tactics.Sym
import Arm.Memory.Separate
```
This isn't using the new VCG stuff, so let's not include `Correctness.ArmSpec`. Also, `Tactics.StepThms` are only used in `#genStepEqTheorems`, which we already have from `AESGCMEncKernelPre`.